extensions/wikia/AdEngine/js/ml/outstream/outstreamLogisticRegression.js

```diff
@@ -23,6 +23,7 @@ define('ext.wikia.adEngine.ml.outstream.outstreamLogisticRegression', [
 		model: linearModel.create(coefficients, intercept),
 		name: 'outstreamlr',
 		wgCountriesVariable: 'wgAdDriverOutstreamLogisticRegressionRabbitCountries',
-		enabled: true
+		enabled: true,
+		cachePrediction: true
```

Comment:
Is is ok, that we cache the prediction here? I'm worried about mobile-wiki and consecutive pageviews. Is the state cleaned on every pageview?

Reply:
:gem: thanks for catching this. Addressed